PSReadLine/Render.cs

```diff
@@ -403,6 +403,14 @@ int PhysicalLineCount(int columns, bool isFirstLogicalLine, out int lenLastPhysi
                 }
 
                 lenLastPhysicalLine = columns % bufferWidth;
+                if(lenLastPhysicalLine == 0)
+                {
+                    // Handle the last column when the columns is equal to n * bufferWidth
+                    // where n >= 1 integers
+                    lenLastPhysicalLine = bufferWidth;
+                    return cnt + (columns - 1) / bufferWidth;
```

Comment:
I find `columns - 1` confusing here - I think `cnt - 1 + (columns / bufferWidth)` is equivalent, and if so, clearer.

Reply:
Updated. thx